src/super_gradients/recipes/training_hyperparams/coco2017_yolox_train_params.yaml

```diff
@@ -0,0 +1,55 @@
+defaults:
+  - default_train_params
+
+max_epochs: 300
+lr_mode: cosine
+cosine_final_lr_ratio: 0.05
+lr_warmup_epochs: 5
+lr_cooldown_epochs: 15
+warmup_bias_lr: 0.0
+warmup_momentum: 0.9
+initial_lr:  0.02
+warmup_mode: yolov5_warmup
+batch_accumulate: 1
+
+save_ckpt_epoch_list: [285]
+
+loss: yolox_loss
+
+criterion_params:
+  strides: [8, 16, 32]  # output strides of all yolo outputs
+  num_classes: 80
+
+loss_logging_items_names: [iou, obj, cls, l1, num_fg, Loss]
+
+optimizer: SGD
+optimizer_params:
+  momentum: 0.9
+  weight_decay: 0.0005  # will be scaled according to effective batch size
+  nesterov: True
+
+ema: True
+mixed_precision: True
+
+valid_metrics_list:
+  - _target_: super_gradients.training.metrics.DetectionMetrics
```

Comment:
don't we have a factory for that?

Reply:
Added to callbacks factory + factory call in SG in https://github.com/Deci-AI/super-gradients/commit/733237b8f40c7d6fbb911edc67d36201d0c131af